README.md

```diff
@@ -1,14 +1,42 @@
-`svgdigitizer.py` can be used to generate xy data of any kind from svg files. It creates data from multiple paths in a single plot.  
+# SVGDigitizer — Create (x,y) Data Points from SVG files.
+
+![Logo](./logo.svg)
+
+The purpose of this project is to digitize plots in scientific publications, recovering the measured data visualized in such a plot.
+
+## Command Line Interface
+
+There's a simple command line interface.
+
+```
+$ python -m svgdigitizer
+Usage: python -m svgdigitizer [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  digitize
+  plot
+$ python -m svgdigitizer plot test/data/Ni111_NaOH_Beden1985_Fig2c.svg
+[displays a plot]
+$ python -m svgdigitizer digitize test/data/Ni111_NaOH_Beden1985_Fig2c
+[creates test/data/Ni111_NaOH_Beden1985_Fig2c.csv]

```

Comment:
I added a simple command line interface to the digitizer. It does not take any options yet but maybe it's a good starting point already for those who do not want to digitize from Jupyter notebooks.